### DISCUSSION TITLE
Change primary monitor completely

### DISCUSSION THREAD

**Discussion by dominictynes** [Noobsai/fullscreen-avoider#8] [Ideas]
Any chance to add an option to have this extension just change the primary monitor completely per the current behavior as opposed to just moving the top bar and, of course, change back when un-fullscreened?

**Comment by Noobsai** [comment]
The mutter doesn't provide any API for setting the primary monitor. Cannot change it from extension

**Reply by dominictynes** [reply]
Ah, ok.  

I had looked at your code,  Then I download Gnome Shell's javascript and found where it was changing primary monitor in the Settings/Display (if you do it there manually) but there was so much redirection/indirection I just couldn't comprehend where, finally, it happens.

Thanks.
